turbinia/workers/__init__.py

```diff
@@ -303,6 +303,10 @@ def execute(
           result.log(msg)
           log.warning(msg)
         else:
+          new_path, _ = self.output_manager.save_local_file(
```

Comment:
Hrm... not sure what the best solution is here.  Since this returns the last path that was saved, if GCS was enabled, this could potentially get the GCS path instead of the local path.  I'll have to take a closer look at this tomorrow, but we might need to refactor save_local_evidence() a it more.

Reply:
FYI, See https://github.com/google/turbinia/pull/319 as a possible fix for this issue.  FWIW I verified it by making sure that the Plaso run was writing to /tmp and the Psort run was operating on the evidence in the permanent output directory.